code/modules/mob/living/carbon/life.dm

```diff
@@ -384,3 +383,36 @@
 		if(360.15 to INFINITY) //360.15 is 310.15 + 50, the temperature where you start to feel effects.
 			//We totally need a sweat system cause it totally makes sense...~
 			bodytemperature += min((body_temperature_difference / BODYTEMP_AUTORECOVERY_DIVISOR), -BODYTEMP_AUTORECOVERY_MINIMUM)	//We're dealing with negative numbers
+/////////
+//LIVER//
+/////////
+
+/mob/living/carbon/proc/handle_liver()
+	var/obj/item/organ/liver/liver = getorganslot("liver")
+	if(((!(NOLIVER in dna.species.species_traits)) && (!liver)) || liver.damage >= 100)

```

Comment:
this might be a dumb question, but will making it into `liver && liver.damage` fix this?